widgets/containers/containerList.widget.js

```diff
@@ -56,6 +59,12 @@ class myWidget extends ListWidget {
     }
   }
 
+  sortContainersByState(state) {
+    this.sortBy = state;
+    this.refreshList();
+  }
+
+  // TODO refactor this
   formatList (containers) {
```

Comment:
I don't love what I had to do to this method. It's unfortunate that the library expects an array instead of an object, because I had to push the status to use it in a calculation later, then pop it.